table/block_based/block_based_table_reader.cc

```diff
@@ -1887,43 +1944,30 @@ CachableEntry<UncompressionDict> BlockBasedTable::GetUncompressionDict(
     std::unique_ptr<const BlockContents> compression_dict_block;
     Status s =
         ReadCompressionDictBlock(prefetch_buffer, &compression_dict_block);
-    size_t usage = 0;
     if (s.ok()) {
       assert(compression_dict_block != nullptr);
       // TODO(ajkr): find a way to avoid the `compression_dict_block` data copy
       dict = new UncompressionDict(compression_dict_block->data.ToString(),
                                    rep_->blocks_definitely_zstd_compressed,
                                    rep_->ioptions.statistics);
-      usage = dict->ApproximateMemoryUsage();
+      const size_t usage = dict->ApproximateMemoryUsage();
```

Comment:
Curious, why adding `const` to `usage`?

Reply:
Just to make the semantics clearer (I don't think the code the compiler generates would be any different in this specific case).